config.json

```diff
@@ -14,5 +14,9 @@
     "so",
     "hr",
     "es"
+  ],
+  "rtlLanguages": [
+    "ar",
+    "fa"
```

Comment:
Didn't the malteser job add a new rtlLanguage?

Reply:
Yes, that new language is added in the malteser locales job pr.